src/api/ledger/transaction.js

```diff
@@ -78,11 +78,22 @@ function getTransactionAsync(identifier: string, options: TransactionOptions,
     }
   }
 
+
+  /* eslint-disable no-unused-vars, handle-callback-err */
```

Comment:
Yeah, you're right about change, but here I really don't care if `getLedgerSequence` returns an error.